client/src/components/buttons/Button.tsx

```diff
@@ -287,11 +293,78 @@ function RoundButtonGroup({ children }: { children: React.ReactNode[] }) {
   return <ButtonGroup className="round-button-group">{children}</ButtonGroup>;
 }
 
+/*
+ * underline Link with icon
+ */
+function UnderlineArrowLink({
+  to,
+  text,
+  tooltip,
+}: {
+  text: string;
+  to: string;
+  tooltip: ReactNode;
+}) {
+  const ref = useRef(null);
+  return (
+    <>
+      <span ref={ref} className={buttonStyles.LinkUnderline}>
+        <Link className="text-decoration-none" to={to}>
+          {text} <ArrowRight />
```

Comment:
This probably looks better:

```suggestion
          {text}
          <ArrowRight className={cx("bi", "ms-1")} />
```